package.json

```
@@ -30,22 +30,22 @@
 		"@typescript-eslint/eslint-plugin": "^8.56.1",
```

Biome (2.4.10)

[error] The key `@typescript-eslint/eslint-plugin` was already declared.

(lint/suspicious/noDuplicateObjectKeys)